tests/Diagnostics.HealthChecks.UnitTests/Composables/HealthCheckTestHelpers.cs

```diff
@@ -87,7 +87,7 @@ public void Log<TState>(LogLevel logLevel, EventId eventId, TState state, Except
 	/// <param name="numberOfFailuresBeforeOk">The number of failures before returning the response.</param>
 	/// <param name="shouldThrowException">Whether the HttpClient should throw an exception.</param>
 	/// <returns>The Http Client factory mock.</returns>
-	internal static Mock<IHttpClientFactory> GetHttpClientFactoryMock(
+	public static Mock<IHttpClientFactory> GetHttpClientFactoryMock(
```

Comment:
They are referenced by the HealthChecks.AspNetCore package. The two packages will eventually be one when we will be able to drop support for .NET Framework, that's why I preferred this approach rather than polluting the configuration solution-wise.

Reply:
We can drop .NET Framework support now...
That's out-of-scope for this PR though.
I don't like opening these up publicly "temporarily" though, as I've no doubt someone will take a dependency on them in their health check tests then we won't be able to make them internal again.
It would be cleaner to add InternalsVisibleTo properties that explicitly specify what can use the internals of this project.